cardano-node/src/Cardano/Node/Tracing/Tracers/Peer.hs

```diff
@@ -81,7 +82,7 @@ ppInFlight f = printf
  (ppMaxSlotNo $ peerFetchMaxSlotNo f)
  (peerFetchReqsInFlight f)
  (Set.size $ peerFetchBlocksInFlight f)
- (peerFetchBytesInFlight f)
+ (getSizeInBytes $ peerFetchBytesInFlight f)
```

Comment:
Why this change?

Reply:
This location too was relying on the orphan instance that is being deleted, and I now need to unpack the data.

Reply:
I think the `PrintfArg` instance should be just moved to `ouroboros-network` if possible, and this place should stay as it was.

Reply:
I agree lets leave this as it is. I don't want to accidentally change the tracing format

Reply:
I reverted the change :+1: